dist/node-connector.js

```diff
@@ -31890,19 +31889,21 @@ module.exports =
 	});
 	exports.default = processColumnarResults;
 	/**
-	   * Because it is inefficient for the server to return a row-based
-	   * data structure, it is better to process the column-based results into a row-based
-	   * format after the fact.
-	   *
-	   * @param {TRowSet} data The column-based data returned from a query
-	   * @param {Boolean} eliminateNullRows A flag that allows removal of null rows from results
-	   * @param {Object} dataEnum A list of types created from when executing {@link #invertDatumTypes}
-	   * @returns {Object} processedResults The formatted results of the query
-	   */
+	 * Because it is inefficient for the server to return a row-based
+	 * data structure, it is better to process the column-based results into a row-based
+	 * format after the fact.
+	 *
+	 * @param {TRowSet} data The column-based data returned from a query
+	 * @param {Boolean} eliminateNullRows A flag that allows removal of null rows from results
+	 * @param {Object} dataEnum A list of types created from when executing {@link #invertDatumTypes}
+	 * @returns {Object} processedResults The formatted results of the query
+	 */
 	function processColumnarResults(data, eliminateNullRows, dataEnum) {
 	  var formattedResult = { fields: [], results: [] };
 	  var numCols = data.row_desc.length;
 	  var numRows = typeof data.columns[0] === "undefined" ? 0 : data.columns[0].nulls.length;
+	  // to satisfy eslint no-magic-numbers rule
+	  var oneThousand = 1000;
```

Comment:
Perhaps `const MILLISECONDS = 1000` or `const MS = 1000` better here to signal intent?

Reply:
thanks for the vigilance, but again this is the built file from the `dist` directory 😛 

comment on the `src` file and I'll make the change there 😄